contracts/KetherSortitionV2.sol

```diff
@@ -118,13 +107,16 @@ contract KetherSortition is Ownable, VRFConsumerBase {
 
   // Internal helpers:
 
-  /**
-   * @notice Only callable by Chainlink VRF, async triggered via startElection().
-   */
-  function fulfillRandomness(bytes32 requestId, uint256 randomness) internal override {
-    require(state == StateMachine.WAITING_FOR_ENTROPY, Errors.NotExecuted);
-    electionEntropy = randomness;
-    state = StateMachine.GOT_ENTROPY;
+  function _getEntropy() view internal returns(uint256) {
```

Comment:
I assume this doesn't pass MEV muster but is good enoygh for testing?

Reply:
Pretty much.